.github/workflows/update-asset-lists.yml

```diff
@@ -0,0 +1,52 @@
+# This is a scheduled workflow that runs daily at 00:00 UTC and can also be triggered manually.
+# To run locally one could use the following command:
+#   act workflow_dispatch -j update-asset-lists --secret-file .env --input source-branch=7859
+#
+# Workflow does the following when triggered:
+# - Runs a script to update asset lists.
+# - Commits and pushes changes to the source-branch ( default: main ).
+# - Opens a PR from the source branch to the target-branch ( default: main ).
+#
+# PR is created using DEFAULT strategy of peter-evans/create-pull-request action:
+# - If there are changes in the source branch, a PR is created.
+# - If there are no changes in the source branch, no PR is created.
+# - If there is an existing PR from the source branch to the target branch, it is updated.
+# - If there is an existing PR from the source branch to the target branch, and there are no changes in the source branch, the PR is closed.
+# See more: https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#action-behaviour
+name: Update Asset Lists
+on:
+  schedule:
+    - cron:  '0 0 * * *'
```

Comment:
Im thinking this should be more like a once a week run max. Honestly I am actually fine with once every 2 weeks. We get assets added regularly and the noise would be a little too much